OpenRA.Game/Traits/DebugPauseState.cs

```diff
@@ -12,6 +12,7 @@
 namespace OpenRA.Traits
 {
 	[Desc("Checks for pause related desyncs. Attach this to the world actor.")]
+	[TraitLocation("World")]
```

Comment:
Style nit: IMO it is cleaner and clearer to keep the `[Desc()]` as the last attribute before the thing its attached to. When going through these to replace strings with the enum, can we also please move these above the descs?

Reply:
As a general rule, definitions (variable and attribute) are best ordered by length. Anything else can be disruptive  when you want to be able to read a piece of code at a glance.

Reply:
Indeed. This is one of the main things behind “cleaner and clearer” (descs are always going to be the longest attribute).